infection.json.dist

```diff
@@ -0,0 +1,17 @@
+{
+    "source": {
+        "directories": [
+            "src"
+        ],
+        "excludes": [
+            "test/**/*.php",
```

Comment:
Do we need to exclude any of these at all? Isn't declaring `source/` sufficient?

Reply:
I honestly don't know whether exclusions are required or not… Removing them completely does not appear to affect the output (65% MSI both with and without). I've removed them now.